<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupData">
     <property name="title">
      <string>Исходные данные</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Балка</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Ширина сечения балки, м:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="input_b">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999999.989999994635582</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Высота сечения балки, м (h):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="input_h">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999999.989999994635582</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Высота сечения балки, м (h&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="input_h0">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999999.989999994635582</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Бетон</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Класс бетона</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="concrete_class">
           <item>
            <property name="text">
             <string>B3,5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B7,5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B12,5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B35</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B40</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B45</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B50</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B55</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B60</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B70</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B80</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B90</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B100</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Толщина защитного слоя бетона, м:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="input_a_s">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999999.989999994635582</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Относительная деформация бетона:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="input_eps_bu">
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="maximum">
            <double>99999999.989999994635582</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Арматура</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Класс арматуры</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Относительная деформация арматуры</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="input_epsilon_s">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999999.989999994635582</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="armature_class">
           <item>
            <property name="text">
             <string>A-I (A240)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A-II (A400)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A-III (A500)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A-IV (A600)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A-V (A800)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A-VI (A1000)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B-I (B500)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bр-I (Bр500)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bр-II (Bр1200)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bр-III (Bр1300)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bр-IV (Bр1400)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bр-V (Bр1500)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bр-VI (Bр1600)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>K-I (K1400)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>K-II (K1500)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>K-III (K1600)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>K-IV (K1700)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Воздействие</string>
        </property>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Изгибающий момент, т∙м&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="input_M">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999999.989999994635582</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Масса, т</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="input_m">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>99999999.989999994635582</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupResult">
     <property name="title">
      <string>Результат</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Напряжение в арматуре, кг/см&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="output_A_s">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Напряжение в растянутой арматуре, кг/см&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="output_sigma_s">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Относительная высота сжатой зоны бетона</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="output_ksi">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Характеристика сжатой зоны бетона&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="output_A_0">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Коэффициент пластичности бетона&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="output_lambda">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Армирование сечения балки, %</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="output_miu">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>slotCalculate()</slot>
 </slots>
</ui>
